modules/wallet/consts.go

```diff
@@ -0,0 +1,12 @@
+package wallet
+
+import (
+	"github.com/NebulousLabs/Sia/types"
+)
+
+var (
+	// DustValue is the value below which a types.Currency is considered Dust.
+	dustValue = func() types.Currency {
+		return types.SiacoinPrecision
+	}
```

Comment:
I would like to have it as a function actually. Though, of the form `func dustValue() types.Currency {`. This is because, unlike for the constants, this is something that I expect to be computed at runtime and to change as the transaction fees change.

Eventually, the transaction fees are going to be calculated dynamically, and when that transition is made the dust outputs should also be determined dynamically.